docs/about/release-notes/migration-faq.md

```diff
@@ -0,0 +1,237 @@
+```
+
+```{dropdown} Can NeMo Curator scale to multi-node, multi-GPU clusters?
+
+Yes. Ray natively orchestrates jobs across nodes and GPUs. For multi-node operations (for example, distributed model training or global deduplication), you'll set up communication (such as NCCL for PyTorch) within the relevant stage's setup logic.
+```
+
+```{dropdown} How does the system manage GPU locality and optimal task placement?
+
+Internally, an allocator ensures data locality for heavy data transfer stages (for example, tasks passing large data between GPU-intensive stages tend to run on the same GPU or node to minimize transfer times). These are handled by Cosmos Xenna or Ray's underlying allocators and optimization routines—not directly inside NeMo Curator itself. These frameworks attempt to colocate data and computation to minimize data transfer and maximize throughput, according to your resource constraints and specifications.
+```
+
+```{dropdown} Can different pipeline stages use different Python environments or dependencies?
+
+Yes, you can specify different Conda environments per stage. Ensure module imports occur inside the stage's process or setup methods (not at the global scope).
+```
+
+---
+
+## Data Processing, Models, and Quality
+
+```{dropdown} How do you handle quality filtering for low-resource languages where no good models exist?
+
+NeMo Curator suggests several strategies (see {ref}`Heuristic Filtering <text-process-data-filter-heuristic>` and {ref}`Quality Assessment <about-concepts-text-data-processing>` for details):
+
+- Use available multilingual models (for example, Qwen, Mistral, or other models with many language capabilities)
+- Annotate high-quality English data with a classifier, translate these data to the target language, and then train a smaller in-language model
+- Upsample high-quality data or downsample and rewrite lower-quality data instead of discarding it (using an LLM rewrite for quality improvement)
+```
+
+```{dropdown} What is the "ensemble model bucketing" workflow?
+
+A set of models assigns a quality score (for example, 0–20), bucketed into high, medium, or low groups. High-quality data might be upsampled or have synthetic variants generated; low-quality data is rewritten rather than simply dropped—especially valuable for low-resource contexts.
+```
+
+```{dropdown} How can we implement or reuse rule-based or custom quality filters? Is YAML (configuration-based) support available?
+
+Quality filters can be implemented as full Python stages in the pipeline (see {ref}`Heuristic Filtering <text-process-data-filter-heuristic>` for available filters and usage examples). YAML or configuration-based filter definitions are available, making it easier to define and reuse filters without writing as much code. Collaboration is encouraged—please contribute region or language-specific filters via pull requests.
```

Comment:
It is true to YAML support is implicitly available. However we have yet not re-added the YAML files that we had in the past (I am aiming to add it for the next release).